# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/PuppetWidget.cpp
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 widget/PuppetWidget.cpp
--- a/widget/PuppetWidget.cpp	Fri Mar 22 18:28:31 2019 +0000
+++ b/widget/PuppetWidget.cpp	Fri Mar 22 18:28:42 2019 +0000
@@ -509,7 +509,7 @@
 
 void PuppetWidget::SetConfirmedTargetAPZC(
     uint64_t aInputBlockId,
-    const nsTArray<ScrollableLayerGuid>& aTargets) const {
+    const nsTArray<SLGuidAndRenderRoot>& aTargets) const {
   if (mTabChild) {
     mTabChild->SetTargetAPZC(aInputBlockId, aTargets);
   }